python/oneflow/test/modules/test_ne.py

```diff
@@ -36,6 +36,8 @@ def _test_ne(test_case, shape, device):
     np_out = np.not_equal(arr1, arr2)
     test_case.assertTrue(np.array_equal(of_out.numpy(), np_out))
     test_case.assertTrue(np.array_equal(of_out2.numpy(), np_out))
+    test_case.assertTrue(input != None, True)
+    test_case.assertTrue(None != input, True)
```

Comment:
```suggestion
    test_case.assertTrue(input != None)
    test_case.assertTrue(None != input)
```
应该是这样吧？assertTrue 应该只接受一个 bool 参数

Reply:
可以了